### DISCUSSION TITLE
Is there a combination store usage method

### DISCUSSION THREAD

**Discussion by 0xbdm0101** [pmndrs/valtio#1027] [Q&A] [answered]
for example：https://pinia.vuejs.org/cookbook/composing-stores.html

a store import b store  ： Basic type （success）
a store import b store  ： reference type （fail）

```js
import { state as localeState } from "./locale";

export const state = proxy({
  // out of sync
  signatureInfo: localeState.wallet_sign_info,
});
```

**Comment by dai-shi** [comment, marked as answer]
In general, Valtio proxies are transparent wrapper around normal JS objects. So, you can do what you can do with JS objects.

For reactive solutions, you might be interested in https://github.com/valtiojs/valtio-reactive.

We also have https://github.com/valtiojs/derive-valtio, but valtio-reactive is a new kid.
